### DISCUSSION TITLE
Components styles missing

### DISCUSSION THREAD

**Discussion by Syrup** [saadeghi/daisyui#3301] [Q&A] [answered]
I am using DaisyUI and React-DaisyUI, and everything worked fine without issues. However, when I was modifying the Tailwind config file, the styles for the Button, Drawer, Navbar, and Stats components disappeared. All I did was modify the Tailwind config file. I have even tried it on another machine, but the issue persists.

What's wrong?

**Comment by Syrup** [comment]
The only change I made in the Tailwind config file was updating the accent color

**Comment by saadeghi** [comment]
Please share a minimal reproduction link.  
It's really hard to debug a project without seeing and running the code.

**Reply by Syrup** [reply]
here https://codesandbox.io/p/github/Syrup/discord-bot-dashboard/master

**Comment by Syrup** [comment, marked as answer]
I figured it out myself! It turns out I just needed to add:
```ts
    'node_modules/daisyui/dist/**/*.js',
    'node_modules/react-daisyui/dist/**/*.js',
```
to the TailwindCSS configuration file, as mentioned in the README of the repository <https://github.com/daisyui/react-daisyui>.
For example:
```ts
module.exports = {
  content: [
    "./app/**/*.{js,jsx,ts,tsx}",
    "node_modules/daisyui/dist/**/*.js",
    "node_modules/react-daisyui/dist/**/*.js",
  ],
  // other configurations...
};
```

**Reply by Syrup** [reply]
I’m not sure why the issue didn’t appear earlier
